.github/workflows/chart-release.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com" 
 
       - name: Install Helm
-        uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 #v3.5
+        uses: azure/setup-kubectl@3e0aec4d80787158d308d7b364cb1b702e7feb7f #v4.0.0
```

Comment:
has this willingly changed from helm to kubectl?

Reply:
I copied it to the wrong place, but I have corrected it